docs/index.md

```diff
@@ -69,7 +69,66 @@ connection "equinix" {
 }
 ```
 
+## Multi-Account Connections
+
+You may create multiple equinix connections:
+
+```hcl
+connection "equinix_dev" {
+  plugin    = "equinix"
+  token     = "XV1KL4QXVHcCYoWT2wbr2NRdRTrZRx3K"
+}
+
+connection "equinix_qa" {
+  plugin    = "equinix"
+  token     = "XV1KL4QXVHcCYoWT2wbr2NRdRTrZRx3L"
+}
+
+connection "equinix_prod" {
+  plugin    = "equinix"
+  token     = "XV1KL4QXVHcCYoWT2wbr2NRdRTrZRx3M"
+}
+```
+
+Each connection is implemented as a distinct [Postgres schema](https://www.postgresql.org/docs/current/ddl-schemas.html). As such, you can use qualified table names to query a specific connection:
+
+```sql
+select * from equinix_qa.equinix_metal_organization
+```
+
+You can multi-account connections by using an [**aggregator** connection](https://steampipe.io/docs/using-steampipe/managing-connections#using-aggregators). Aggregators allow you to query data from multiple connections for a plugin as if they are a single connection.
+
+```hcl
+connection "equinix_all" {
+  plugin      = "equinix"
+  type        = "aggregator"
+  connections = ["equinix_dev", "equinix_qa", "equinix_prod"]
+}
+```
+
+Querying tables from this connection will return results from the `equinix_dev`, `equinix_qa`, and `equinix_prod` connections:
+
+```sql
+select * from equinix_all.equinix_metal_organization
+```
+
+Alternatively, can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphbetically, so that it is the first connection in the search path (i.e. `equinix_all` comes before `equinix_dev`):
```

Comment:
```suggestion
Alternatively, you can use an unqualified name, which will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphabetically so that it is the first connection in the search path (i.e. `equinix_all` comes before `equinix_dev`):
```